include/tlog/errs.h

```diff
@@ -141,4 +141,70 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to the "perrs" stack, with message as a constant string
+ * and then go to cleanup.
+ *
+ * @param _msg   The message string to copy and put into the pushed error.
+ *              Cannot be NULL.
+ */
+#define TLOG_ERRS_RAISES(_msg) \
+     do {                                                                 \
+         fprintf(stderr,"%s:%s:%d\n", __FILE__, __FUNCTION__, __LINE__);  \
```

Comment:
First, this change shouldn't be a part of this commit, or PR. Then, it goes against the idea of the "error stack", part which is to keep errors until a later time, when we can meaningfully output them. E.g. when the terminal we run under is in raw mode, the messages we print will not print nicely. Another part is that the caller decides when to output those error messages. With your change, the source code information will be printed immediately, no matter if the caller and the consumer of the error stack would even want to output it at all.

The source code information should be incorporated into the error stack itself, and attached to each error message, the item in the stack. It might be a good exercise to figure out how to best do that. Still, it shouldn't be a part of this PR.